rcon/game_logs.py

```diff
@@ -151,6 +151,8 @@ def run(self, loop_frequency_secs=2, cleanup_frequency_minutes=10):
                 last_cleanup_time = datetime.datetime.now()
 
             dp = self.rcon.get_detailed_players()
+            if dp['fail_count'] > 0:
```

Comment:
do we want to do a .get() here in case the key is missing?

Reply:
not really necessary. I think this one is always set :)